docs/DocGenerator/DocGenerator.py

```diff
@@ -12,24 +12,25 @@
 import json
 import yaml
 import ast
-from Config import Config
```

Comment:
We should remove the empty __init__.py file.

Reply:
The init.py file is required to convert the lib folder into a python module capable to be imported by the DocGenerator.py module.

Reply:
Good to know that, I thought it was only needed a lib folder.